x/mint/keeper/genesis.go

```diff
@@ -6,17 +6,31 @@ import (
 	"github.com/osmosis-labs/osmosis/v7/x/mint/types"
 )
 
+const developerVestingAmount = 225_000_000_000_000
+
 // InitGenesis new mint genesis.
 func (k Keeper) InitGenesis(ctx sdk.Context, ak types.AccountKeeper, bk types.BankKeeper, data *types.GenesisState) {
+	if data == nil {
+		panic("nil mint genesis state")
+	}
```

Comment:
Just for my learning, is there a reason you explicitly panic for a nil data input but not for any of the other inputs?

Reply:
The keeper inputs should get removed

Reply:
Good catch, I don't know why I didn't see it